app/hooks/useQuestionStateInUrl.ts

```diff
@@ -200,7 +200,7 @@ export default function useQuestionStateInUrl(minLogo: boolean, initialQuestions
       const onSiteAnswers = onSiteAnswersRef.current
       if (onSiteAnswers.length === 0 && relatedQuestions.length > 0) {
         // if onSiteAnswers (needed for relatedQuestions) are not loaded yet, wait a moment to re-run
-        setTimeout(() => toggleQuestion(questionProps, options), 500)
+        setTimeout(() => toggleQuestion(questionProps, options), 200)
```

Comment:
should this have some kind of back-off mechanism? Or at least only try n times?

Reply:
good question, but it's not doing any requests, so even if there is a problem and `onSiteAnswers` are never loaded, doing an `if` comparison every 200ms won't fry any eggs => not a problem IMHO